numpy/ma/core.py

```diff
@@ -3091,6 +3091,14 @@ def view(self, dtype=None, type=None, fill_value=None):
         type : Python type, optional
             Type of the returned view, either ndarray or a subclass.  The
             default None results in type preservation.
+        fill_value : scalar, optional
+            The value to use for invalid entries (None by default).
+            If None, the `fill_value` attribute of the array is used instead
+            or the default `fill_value` of the `dtype`.
```

Comment:
Worth clarifying that the original fillvalue is only used if no dtype argument is passed.

Reply:
This is also mentioned in the Notes section. I wanted to avoid longish explanations in the param description if they are also present shortly artet.

Reply:
How about _"if `None`, then the fill-value is inferred from the passed dtype, or in its absence the original array"_?

Reply:
I like it very much - it's short and more accurately describes the behavior. 

I added that to the commit.